client/src/components/LinkPicker/LinkPicker.scss

```
@@ -105,6 +105,10 @@
   &--published::before {
     display: none;
   }
+
+  &.readonly {
```

```suggestion
  &--readonly {
```
Safer to use a BEM class rather than a generic/global `readonly` that may accidentally get values set from somewhere else

You'll need to update this is bunch of places in this PR

---

Updated